packages/table/src/utils/getDataPoints.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Should we be exporting these from core?

Reply:
yes. export it from core and removed the duplications in table.